echopype/convert/parse_ad2cp.py

```diff
@@ -315,13 +315,13 @@ def _postprocess(self, field_name):
 
         if self.data_record_format == self.BURST_AVERAGE_VERSION2_DATA_RECORD_FORMAT:
             if field_name == "configuration":
-                self.data["pressure_sensor_valid"] = self.data["configuration"] & 0b0000_0000_0000_0001
-                self.data["temperature_sensor_valid"] = self.data["configuration"] & 0b0000_0000_0000_0010
-                self.data["compass_sensor_valid"] = self.data["configuration"] & 0b0000_0000_0000_0100
-                self.data["tilt_sensor_valid"] = self.data["configuration"] & 0b0000_0000_0000_1000
-                self.data["velocity_data_included"] = self.data["configuration"] & 0b0000_0000_0010_0000
-                self.data["amplitude_data_included"] = self.data["configuration"] & 0b0000_0000_0100_0000
-                self.data["correlation_data_included"] = self.data["configuration"] & 0b0000_0000_1000_0000
+                self.data["pressure_sensor_valid"] = min(1, self.data["configuration"] & 0b0000_0000_0000_0001)
+                self.data["temperature_sensor_valid"] = min(1, self.data["configuration"] & 0b0000_0000_0000_0010)
+                self.data["compass_sensor_valid"] = min(1, self.data["configuration"] & 0b0000_0000_0000_0100)
+                self.data["tilt_sensor_valid"] = min(1, self.data["configuration"] & 0b0000_0000_0000_1000)
+                self.data["velocity_data_included"] = min(1, self.data["configuration"] & 0b0000_0000_0010_0000)
+                self.data["amplitude_data_included"] = min(1, self.data["configuration"] & 0b0000_0000_0100_0000)
+                self.data["correlation_data_included"] = min(1, self.data["configuration"] & 0b0000_0000_1000_0000)
```

Comment:
Could you explain why we need `min(1, parsed_values)` here? If you shift the bits, wouldn't the parsed values always be 1?

Reply:
I did min here instead of shifting, I should probably shift instead